cmd/pebble/cmd_boot_firmware_common.go

```diff
@@ -0,0 +1,55 @@
+//go:build !termus
+// +build !termus
+
+// Copyright (c) 2023 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"errors"
+
+	"github.com/jessevdk/go-flags"
+)
+
+type cmdBootFirmware struct {
+	clientMixin
+
+	Force   bool `long:"force"`
+	Verbose bool `short:"v" long:"verbose"`
+}
+
+var bootFirmwareDescs = map[string]string{
+	"force":   `Skip all checks`,
+	"verbose": `Log all output from services to stdout`,
+}
+
+var shortBootFirmwareHelp = `Bootstrap a system with Pebble running as PID 1`
+
+var longBootFirmwareHelp = `
```

Comment:
There is a block of common code duplicated between *_common and *_termus. Would it not be cleaner and leaner to have a common file with only the help command stuff, and then two variants with different Executes ?

Reply:
It also feels a bit strange that the common case describes a scenario in the help it does not support, in the same file.

Reply:
At first I went with a similar approach but ended up with a lot of boilerplate and extra files with no real content. As far as I understand, it is a convention in Golang to prefer a bit of redundancy instead of more boilerplate.

Reply:
Also, I don't think it is a good idea to have help messages change conditionally. The idea is not to treat the commands as different, but to treat the unsupported scenarios as run-time errors during the execution of the command that's "unique" to the user (even if it's two commands technically)